tests/core/abi-utils/test_abi_utils.py

```diff
@@ -167,9 +1073,452 @@ def test_event_abi_to_log_topic(event_abi, expected):
             "Transfer(address,address,uint256)",
             "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
         ),
+        (
+            "Transfer(address,address,uint256)",
```

Comment:
This test case is a duplicate of the one above it.

Reply:
Morphed this into a new test case, good catch!